src/test/java/org/kainos/ea/validator/JobRoleRequestValidatorTest.java

```diff
@@ -0,0 +1,128 @@
+package org.kainos.ea.validator;
+
```

Comment:
there are no tests to check the invalid lengths of **job title, link, responsibilities and requirements** fields